test/xml/base64.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is the sample XML generated by http://xsd2xml.com/ with some minor changes to workaround #13. Currently, there aren't any tests that actually use this file, but I think it would be good to ensure that all the generated code can actually be used to unmarshal this file.